source/grid/manifold.cc

```diff
@@ -671,10 +700,9 @@ FlatManifold<dim, spacedim>::get_new_points(
           }
   if (adjust_periodicity == true)
     {
-      modified_points.resize(surrounding_points.size());
-      std::copy(surrounding_points.begin(),
-                surrounding_points.end(),
-                modified_points.begin());
+      modified_points.insert(modified_points.end(),
+                             surrounding_points.begin(),
+                             surrounding_points.end());
```

Comment:
Isn't this equivalent to just saying
```
  modified_points = surrounding_points;
```
or if the types cannot be copied this way, 
```
  modified_points = boost::container::small_vector<Point<spacedim>, 200>(surrounding_points.begin(), surrounding_points.end());
```
?